src/zswag_client/spec.py

```diff
@@ -41,20 +38,12 @@ class ZserioSwaggerSpec:
 
     def __init__(self, spec_url_or_path: str):
         spec_url_parts = urlparse(spec_url_or_path)
-        extension = "yaml"
-        if "." in spec_url_parts.path:
-            extension = os.path.splitext(spec_url_parts.path)[1].lower()
-        if spec_url_parts.scheme in {"http", "https"}:
+        if spec_url_parts.scheme in {"http", "https", "file"}:
```

Comment:
Python requests module has no default connection adapter for URI scheme 'file'.
```suggestion
        if spec_url_parts.scheme in {"http", "https"}:
```

Reply:
Good catch 🐠